apps/intro/pages/index.tsx

```diff
@@ -75,7 +76,7 @@ export default function Index() {
           variant="small"
           className="fixed bottom-12 text-[13.5px] text-opacity-30 font-normal"
         >
-          Synonym Software Ltd. ©2024.
+          Synonym Software Ltd. ©{currentYear || 2025}.
```

Comment:
```suggestion
          Synonym Software Ltd. ©{currentYear}.
```
The fallback || 2025 is unnecessary because new Date().getFullYear() always returns a valid year.